pkg/monitoring/alloy/configmap.go

```diff
@@ -181,10 +186,15 @@ func (a *Service) generateAlloyConfig(ctx context.Context, cluster *clusterv1.Cl
 		Tenants:         tenants,
 		DefaultTenantID: commonmonitoring.DefaultWriteTenant,
 
-		QueueConfigCapacity:          commonmonitoring.QueueConfigCapacity,
```

Comment:
Does it need to be updated a similar way in `pkg/monitoring/prometheusagent/secret.go`?